articles/cms/wordpress/jwt-authentication.md

```diff
@@ -53,3 +53,15 @@ Vydcvl6gpWNeUE
 
 - [Cookies vs Tokens. Getting auth right with Angular.JS](https://auth0.com/blog/2014/01/07/angularjs-authentication-with-cookies-vs-token/)
 - [10 Things You Should Know about Tokens](https://auth0.com/blog/2014/01/27/ten-things-you-should-know-about-tokens-and-cookies/)
+
+## Keep Reading
```

Comment:
Rather than repeating this lots of times in 5 files, can it be turned into an include? I don't really mind if the current section is included in the next-steps?

Reply:
@cocojoe - I'll add it to my list for the next go-around, I have a number of things that need improving. 

OK to approve for now?

Reply:
I guess

Reply:
For future reference, we don't include in one doc links that point to the same doc. We have them reported back as broken links (it has happened more than once). So if you do use a doc fragment at some point make sure that the same doc does not list itself.